hack/tools/release/notes.go

```diff
@@ -307,8 +309,11 @@ func run() int {
 		}
 	}
 
+	if *releaseCandidate {
+		fmt.Printf("🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).\n")
+	}
+
 	if *addKubernetesVersionSupport {
-		// TODO Turn this into a link (requires knowing the project name + organization)
```

Comment:
This comment didn't make sense to me, based on the [original intention](https://github.com/kubernetes-sigs/cluster-api/commit/d76ea10ffbc9ab697f56ece40ec6b67ca252f423#diff-818b0767917bf0415c71e90e83d21b6faf07a4096264d71de9646f2c1ab4f48aR123-R124) it seemed to be better placed with `## Changes since`, so I moved it

Reply:
I think we can remove it? the `repo` variable contains this info I believe

Reply:
+1 to remove it